docs/learn/about-iota/data-transfer.md

```diff
@@ -1,34 +1,196 @@
 ---
-id: data-transfer
 title: Data Transfer
 description: A major use-case for the IOTA Technology is Data Transfer. Learn how it works.
 ---
 
 
-# Heading 1 (Page Title)
+# Data Transfer
 
-content - introduction to the topic
+IOTA is the only Cryptocurrency and DLT Technology that is capable of transferring Data totally free, fast, immutable, unforgeable, and secure. 
+These unique feautures open up a wide array of use cases that normal cryptocurrencies can never support.
```

Comment:
```suggestion
These unique features open up a wide array of use cases that normal cryptocurrencies can never support.
```

Again a bold statement that we might want to avoid. My opinion would be to base our information on long lasting facts, rather than current state that might change quickly.

Reply:
removed and rewritten